File: Back/src/SistemaCompra.Domain/Produto.cs
using System.Collections.Generic;

namespace SistemaCompra.Domain
{
    public class Produto
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public string UnidMedida { get; set; }
        public int FamiliaProdutoId { get; set; }
        public FamiliaProduto FamiliaProduto { get;  }
        public IEnumerable<SolicitacaoProduto> SolicitacaoProdutos { get;}
    }
}

File: Front/ProEventos-App/src/app/models/Produto.ts
import { FamiliaProduto } from "./FamiliaProduto";
import { SolicitacaoProduto } from "./SolicitacaoProduto";

export class Produto {
   id: number;
   descricao: string;
   unidMedida: string;
   familiaProdutoId: number;
   descricaoFamilia: string;
   familiaProduto: FamiliaProduto;
   solicitacaoProdutos: SolicitacaoProduto[];
}
